src/natcap/invest/utils.py

```diff
@@ -583,8 +583,12 @@ def read_csv_to_dataframe(
     # allow encoding kwarg to override this if it's provided
     if not encoding and has_utf8_bom(path):
         encoding = 'utf-8-sig'
```

Comment:
If `encoding` isn't specified, should we default to using `encoding = 'utf-8'`? It seems like that would be the `pandas` default, but it isn't clear from the [docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html) what happens when `encoding=None`.

Reply:
If `encoding` is not specified then we do let pandas do it's own thing (offhand I'm not sure what that is either). But in the rare case where the file has a BOM, I guess pandas does not handle the BOM correctly when encoding=None, hence the need to identify the BOM first and specify encoding as `utf-8-sig`.

Reply:
Maybe we should be explicit about assuming UTF8 when there isn't a BOM too? something like 
```
if not encoding:
    if has_utf8_bom(path):
        encoding = 'utf-8-sig'
    else:
        encoding = 'utf-8'
```